paddle/fluid/operators/mkldnn/conv_mkldnn_op.cc

```diff
@@ -411,10 +411,17 @@ class ConvMKLDNNHandlerT
 
     bool is_multi_channel = scale_weights_data.size() > 1;
     int mask_reorder = is_multi_channel ? 1 << 0 : 1;
-    const int count =
-        is_multi_channel
-            ? (groups > 1 ? (weights_tz)[1] * (weights_tz)[0] : (weights_tz)[0])
-            : 1;
+
+    int count;
```

Comment:
Maybe if you initiate with :
`int count =1;`
then
```
else {
      count = 1;
    }
```
won't be necessary. What is your opinion ?

Reply:
Ok. I incorporated your suggesstion while keeping if-else blocks as requested by @jakpiase